examples/resources/spectrocloud_cluster_aws/resource.tf

```diff
@@ -69,14 +70,26 @@ resource "spectrocloud_cluster_aws" "cluster" {
     count                   = 1
     instance_type           = "t3.large"
     disk_size_gb            = 62
-    azs                     = ["us-west-2a"]
+    azs                     = ["us-east-2a"]
+    az_subnet {
+        id = "subnet-08864975df862eb58"
+        az = "us-east-2a"
+      }
+    az_subnet {
+        id = "subnet-031a7ff4ff5e7fb9a"
+        az = "us-east-2a"
+      }
   }
 
   machine_pool {
     name          = "worker-basic"
     count         = 1
     instance_type = "t3.large"
-    azs           = ["us-west-2a"]
+    azs           = ["us-east-2a"]
```

Comment:
`azs` and `az_subnets` are mutually exclusive. An example should have `azs` for dynamic provisioning and `az_subnets` for static provisioning